src/main/scala/io/archivesunleashed/app/ExtractEntities.scala

```diff
@@ -21,27 +21,35 @@ import io.archivesunleashed.matchbox.{NER3Classifier, RemoveHTML}
 import org.apache.spark.SparkContext
 import org.apache.spark.rdd.RDD
 
-/**
-  * Extracts entities
+/** Conducts Named Entity Recognition (NER) on a WARC or ARC file.
+  *
+  * [[http://nlp.stanford.edu/software/CRF-NER.html Named Entity Recognition]]
+  * applies rules formed in a [[https://stanfordnlp.github.io/CoreNLP  Named
+  * Entity Classifier]] to identify locations, people or other objects from data.
   */
 object ExtractEntities {
 
-  /**
-    * @param iNerClassifierFile path of classifier file
+  /** Extracts NER entities from WARC or ARC files at a given path to a given output directory.
+    *
+    * @param iNerClassifierFile path to NER classifier file
     * @param inputRecordFile path of ARC or WARC file from which to extract entities
     * @param outputFile path of output directory
+    * @param sc the Apache Spark context
+    * @return an rdd with classification entities.
     */
   def extractFromRecords(iNerClassifierFile: String, inputRecordFile: String, outputFile: String, sc: SparkContext): RDD[(String, String, String)] = {
     val rdd = RecordLoader.loadArchives(inputRecordFile, sc)
       .map(r => (r.getCrawlDate, r.getUrl, RemoveHTML(r.getContentString)))
     extractAndOutput(iNerClassifierFile, rdd, outputFile)
   }
 
-  /**
+  /** Extracts NER entities from tuple-formatted derivatives scraped from a website.
```

Comment:
NER stands for Named Entity Recognition, so either say "Extracts named entities" or "Performs NER"...